src/test/java/OrderSmokeTests.java

```diff
@@ -1,50 +1,59 @@
 import controller.OrderController;
-import io.restassured.response.Response;
-import models.Order;
-import org.testng.Assert;
+import io.qameta.allure.Description;
+import io.qameta.allure.testng.AllureTestNg;
+import org.apache.http.HttpStatus;
+import org.testng.annotations.Listeners;
 import org.testng.annotations.Test;
-import static utils.Constants.*;
 
+import static utils.Constants.DEFAULT_ORDER;
+import static utils.Constants.NON_EXIST_ID_INT;
+
+@Listeners({AllureTestNg.class})
 public class OrderSmokeTests extends BaseTest {
 
     OrderController orderController = new OrderController();
 
     @Test
```

Comment:
To add readable name in Allure in addition of @Description add name inside @Test tag. 
Example: @Test(description = "Add order test with default order")